action/action-simulator/src/main/java/com/powsybl/action/simulator/loadflow/CaseExporter.java

```diff
@@ -36,27 +34,22 @@ public CaseExporter(Path outputCaseFolder, String outputCaseFormat) {
     }
 
     @Override
-    public void loadFlowDiverged(Network network, Contingency contingency, int round) {
-        exportNetwork(network, contingency, round);
+    public void loadFlowDiverged(Contingency contingency, Network network, int round) {
+        exportNetwork(contingency, network, round);
     }
 
     @Override
-    public void loadFlowConverged(Network network, Contingency contingency, int round, List<LimitViolation> violations) {
-        exportNetwork(network, contingency, round);
+    public void loadFlowConverged(Contingency contingency, List<LimitViolation> violations, Network network, int round) {
+        exportNetwork(contingency, network, round);
     }
 
-    private void exportNetwork(Network network, Contingency contingency, int round) {
+    private void exportNetwork(Contingency contingency, Network network, int round) {
         if (null != contingency && StringUtils.isNotEmpty(contingency.getId())) {
             String contingencyIdAndRound = contingency.getId().replace(' ', '_').replace('.', '_') + String.valueOf("_Round_" + round);
-            Exporters.export(outputCaseFormat, network, new Properties(), outputCaseFolder.resolve(contingencyIdAndRound + "_" + network.getId() + "_" + dateForName() + "." + outputCaseFormat + ".gz"));
+            Exporters.export(outputCaseFormat, network, new Properties(), outputCaseFolder.resolve(contingencyIdAndRound + "_" + network.getId() + "_" + "." + outputCaseFormat + ".gz"));
         } else {
-            Exporters.export(outputCaseFormat, network, new Properties(), outputCaseFolder.resolve("N_situation" + "_" + network.getId() + "_" + dateForName() + "." + outputCaseFormat + ".gz"));
+            String nSituationAndRound = "N_situation" + String.valueOf("_Round_" + round);
+            Exporters.export(outputCaseFormat, network, new Properties(), outputCaseFolder.resolve(nSituationAndRound + "_" + network.getId() + "_" + "." + outputCaseFormat + ".gz"));
```

Comment:
Refactor this method: the if/else block should only used to generate the filename. Extract the Exporters.export call from this block